--- posix/lposix.c	2003-11-05 16:23:48.000000000 -0800
+++ posix/lposix.c.new	2008-07-17 11:37:09.000000000 -0700
@@ -599,7 +599,7 @@
  clock_t elapsed;
 };
 
-#define pushtime(L,x)		lua_pushnumber(L,((lua_Number)x)/CLK_TCK)
+#define pushtime(L,x)		lua_pushnumber(L,((lua_Number)x)/CLOCKS_PER_SEC)
 
 static int Ftimes(lua_State *L, int i, const void *data)
 {
@@ -620,7 +620,7 @@
 	"utime", "stime", "cutime", "cstime", "elapsed", NULL
 };
 
-#define storetime(L,name,x)	storenumber(L,name,(lua_Number)x/CLK_TCK)
+#define storetime(L,name,x)	storenumber(L,name,(lua_Number)x/CLOCKS_PER_SEC)
 
 static int Ptimes(lua_State *L)			/** times() */
 {
